src/pages/word-duel.jsx

```diff
@@ -1,15 +1,8 @@
-import React, { useState } from "react";
-import './word-duel.css';
+import React, { useState, useEffect } from "react";
 import Keyboard from '../components/keyboard';
 import Board from '../components/board';
-
-const MAX_GUESSES = 6;
-
-const EvaluationColor = {
-    Gray: "gray",
-    Yellow: "yellow",
-    Green: "green"
-}
+import * as util from '../utils/utils';
```

Comment:
You could also consider doing `import { someFunc } from '../utils';`

but also renaming `utils.js` to `index.js` so you can simplify your import (like above)